### PR TITLE
Remove Slack, use Auth0, use different configurations, redesign user form, remove non needed functions + files

### PR DESCRIPTION
**Pull request by NickWalt01**

### CODE REVIEW

File: landing_page_app/main/scripts/github_script.py [hunk 1]
```diff
@@ -19,64 +16,37 @@ class GithubScript:
...
-                    pre_approved = True
-            elif organisation.lower() == MINISTRY_OF_JUSTICE:
-                if email_domain in MOJ_ORG_ALLOWED_EMAIL_DOMAINS:
-                    pre_approved = True
-        return pre_approved
+    def add_returning_user_to_github_org(self, username: str, organisations: list):
+        if username == "" or username is None or len(organisations) == 0:
+            logger.debug(
+                "add_returning_user_to_github_org: incorrect function argument"
+            )
```
Comment (connormaglynn) [inline landing_page_app/main/scripts/github_script.py:23]: Think it's worth throwing an error here so the user is redirected to an error page?

Comment (NickWalt01) [reply landing_page_app/main/scripts/github_script.py:23]: The function before this function check the input parameters are correct:

if current_app.github_script.validate_user_rejoining_org(selected_orgs, username):

**Review by connormaglynn — APPROVED** [review]
LGTM 🚀

File: landing_page_app/main/scripts/github_script.py (hunk 1 above)
Comment (NickWalt01) [reply landing_page_app/main/scripts/github_script.py:23]: which redirects to the error screen:

return error(
    "Username not found or has expired. Create a new request and leave the username box empty."
)

**Pull request merged**